Helenyin155 patch 1

helenyin155 opened the pull request.

helenyin155 committed:
Create choosing-major-by-money-or-passion.mdx

helenyin155 committed:
Update choosing-major-by-money-or-passion.mdx

helenyin155 committed:
Create choosing-your-major-by-money-or-passion.mdx

helenyin155 committed:
Create med-sci-year-1

helenyin155 committed:
Delete choosing-major-by-money-or-passion.mdx

helenyin155 committed:
Create the-world-the-chaos-and-the-high-school-student.mdx

helenyin155 committed:
Create how-to-be-proactive-in-high-school.mdx

helenyin155 committed:
Update how-to-be-proactive-in-high-school.mdx

helenyin155 committed:
Updated the summary

hirundinidae commented on _digest/october-2020/how-to-be-proactive-in-high-school.mdx:
```diff
@@ -0,0 +1,26 @@
+    <p><br>As mentioned earlier, admissions officers don’t necessarily consider the overachieving biology major with research publications in high school over everyone else, because in most cases, this kind of achievement is the result of individual opportunity and resources early on. However, they do value a student’s assertiveness to gain practical knowledge. Whether this means engaging in summ ...
+  </ol>
+  <p>Sources cited:</p>
+  <p>https://medapplications.com/autobiographical-sketch-for-medical-school-medapplications/</p>
```
This should be a link (using `<a>` tags)

hirundinidae commented on _digest/october-2020/med-sci-year-1:
```diff
@@ -0,0 +1,45 @@
+---
```
Rename file to include .mdx ending

xsharonhe replied:
This is mandatory ^^

hirundinidae commented on _digest/october-2020/choosing-your-major-by-money-or-passion.mdx:
```diff
@@ -0,0 +1,32 @@
+  <p>Choosing a major can be one of the most intimidating decisions a young adult can make. It’s quite an investment: several years of consistent schooling, an upwards of thirty thousand dollars in tuition, and numerous sleepless nights for a major that you may not be completely sold on. It’s a scary thought. For many – myself included – it’s a battle between what will ensure a steady income, and ...
+  
+  <p>According to bestcolleges.com, when choosing a major, one should consider the following:</p>
+  <li> <u>Overall program cost</u> </li>
+  <li> <u>Salary expectations</u> </li>
+  <li> <u>Employment rates in the field</u> </li>
+  <li> <u>Advanced degree opportunities</u> </li>
```
Wrap in a `<ul>` or `<ol>` (use list item tags inside of a list)

hirundinidae reviewed (commented):
I can't actually view these on the blog locally, I think because the db is being changed. For now, the HTML looks good + renders as a markdown file, just a couple of things to change.

xsharonhe commented on _digest/october-2020/choosing-your-major-by-money-or-passion.mdx:
```diff
@@ -0,0 +1,32 @@
+  
+  <ol>
+    <li> <u><strong><em>“</em></strong><strong><em>The process of receiving or giving systematic instruction, especially at a school or university.”</em></strong> This definition is what most of us think of. It’s the definition that was fed to us by our parents and school. The phrase “systematic instruction” has a negative undertone – as if students don’t have control over our learning, which is  ...
+    <li> <u><strong> </strong><strong><em>“An enlightening experience.”</em></strong> I believe this terminology should be reflected more into our schooling. Regardless of what field you end up pursuing, education can, and should be, an illuminating and uplifting experience.</u> </li>
```
The first strong opening and closing tag seems wrong. It's redundant and not needed since it's not encompassing anything

helenyin155 committed:
Rename med-sci-year-1 to med-sci-year-1.mdx

xsharonhe commented on _digest/october-2020/choosing-your-major-by-money-or-passion.mdx:
```diff
@@ -0,0 +1,32 @@
+  <p>I understand that often, people don’t have the privilege of getting an education in something they truly enjoy because they don’t have the space for uncertainty. However, I simply want to shed light on another definition of this word that has developed a negative connotation. </p>
+  <p>People tend to neglect taking the arts or humanities because it typically doesn’t make the most money, and that’s a valid point. However, because of this, less and less people are getting educated about the society we live in, and more and more people don’t understand how politics, relationships, and culture work. As a result, there is a giant socio-political divide in our modern society tha ...
+  <p>“You can fail at what you don’t want, so you might as well take a chance on doing what you love.” – Jim Carrey</p>
+  <p><strong>*This article is in no way telling you what you should and shouldn’t do. It’s just another viewpoint to take note of.</strong></p>
```
Can we remove the asterisks here? I think it might mess with the rendering of these files

xsharonhe reviewed (commented):
Address the comments and rerequest for review so I can see if it renders (since database works fine for me)

helenyin155 committed:
Fixed link

helenyin155 committed:
Update choosing-your-major-by-money-or-passion.mdx

helenyin155 committed:
Got rid of unnecessary strong tag

helenyin155 committed:
Removed astericks

hirundinidae committed:
UAS 3 step (#39)

* Add card styling

* Resolve non-boolean attribute prop error

* Fix children with same key

* Add conditional card styling

* Change margin and breakpoints

* Add title and buttons, decrease margin

* Add color to 3 step title

* Add comments for student resource

* Adjust title font and size

* Increase card margin and remove button

jilly477 committed:
Add opportunities backend (#40)

* attempted to create backend for opportunities page. pushed to inspect prisma database error

* Commit changes

* pretty formatting

* fix package-lock.json merge conflicts

* delete package-lock,json

Co-authored-by: xsharonhe <xsharonhe@gmail.com>

xsharonhe committed:
Modify package-lock.json

xsharonhe committed:
Add profiles (#41)

* Add profiles

* New workflow

* Add more board members

* Add loading component to blog

xsharonhe committed:
Add database changes to suit filtering on cities

xsharonhe committed:
Add changed cache

realsimonwu committed:
Fix issues with adding the background graphic and then merge (#22)

* Add background graphic and some small changes to `TopSection`

Added the background graphic file to `public/home`, and made some small changes to `TopSection` to accomodate for the top graphic. Note that this still has a lot of issues right now at small viewport breakpoints that I'm trying to resolve.

* Fix the gap that appears at small breakpoints

* Run the linter on everything

* Update TopSection.jsx

* Update TopSection.jsx

* Create the `SectionWrapper` component

I have created the `SectionWrapper` component. As of now, to pass images to be used as backgrounds the image paths have to be relative to `/public/` - this means that the ordinary method of using `import` won't work.

It still needs some work: I may try to incorporate props that will allow for fine control of the positioning of the background image. Personally, I think it will probably be better in the long run to remake the background blobbies that already exist to accommodate already-created components, but the addition of those fine positioning controls will probably still be beneficial.

As of right now it looks TERRIBLE with pre-existing components and background blobbies. Other components will require modification (e.g. I intend to fix `TopSection` to accommodate this new background component).

* Add in parameters for vertical and horizontal offsets

The parameters `verticalOffset`, `horizontalOffset`, `mobileVerticalOffset` and `mobileHorizontalOffset` have been added, allowing for fine positioning control of the `background-image`

* Update index.jsx

* Update TopSection.jsx

* Update index.jsx

* Prettier fixes

xsharonhe committed:
Add carousel testimonials, sponsors, contact page (#42)

* Add profiles

* New workflow

* Add more board members

* Add loading component to blog

* Add external link to student resource

* Fix board carousel

* Finish contact page

* Add email contact and fix media

* Finish adding logos

* Testimonials complete

* Prettier fixes

* Last prettier change

* Update UAS3Step.jsx

Change Link to a for external Discord invite

* Small nitpicks fixed

* Add some background images

* Prettier fixes

* Suggested fixes

xsharonhe committed:
Merge branch 'helenyin155-patch-1' of github.com:PuMProfessionals/pump-web-v2 into helenyin155-patch-1

xsharonhe commented on _digest/october-2020/how-to-be-proactive-in-high-school.mdx:
```diff
@@ -0,0 +1,26 @@
+    <li> <u> <strong>Continue pursuing a wide range of interests</strong> </u> </li>
+    <p><br>Considering that you want to pursue a highly-rigorous line of work, you likely immerse yourself in extensive co-curricular activities. Although it may be difficult to maintain a variety of commitments, especially into the latter years of your secondary education, consider remaining involved, even if that means attending less frequently in weeks of heavy course-load. Comparatively, if y ...
+    <li> <u> <strong>Although your co-curricular activities need not concern medicine directly, show your passion for science in whatever you undertake</strong> </u> </li>
+    <p><br>In recent admissions rounds, institutions have duly noted that a medical student does not have one appearance and that the community of physicians is just as diverse as the field of medicine. In light of this, it is imperative that students pursue whatever subjects are of interest to them, from political science to engineering. The wide range of clubs and societies in most school commu ...
```
It should be ```<br />``` all HTML elements have self-closing tags

xsharonhe reviewed (commented):
Everything else works fine but last parsing error. Sorry about late review I didn't see this earlier

Edit: sorry I had to push for you because it was breaking prod database. It'll be like that until I can add a feature

xsharonhe committed:
Change line breeak
